docs/docsite/rst/getting_started/get_started_inventory.rst

```diff
@@ -6,44 +6,40 @@ Building an inventory
 
 Inventories organize managed nodes in centralized files that provide Ansible with system information and network locations.
 Using an inventory file, Ansible can manage a large number of hosts with a single command.
-Inventories also help you use Ansible more efficiently by reducing the number of command-line options you need to specify.
-For example, inventories usually contain the SSH user so you do not need to include the ``-u`` flag when running Ansible commands.
 
-In the previous section, you added managed nodes directly to the ``/etc/ansible/hosts`` file.
-Now let's create an inventory file that you can add to source control for flexibility, reuse, and for sharing with other users.
+To complete the following steps, you will need the IP address or fully qualified domain name (FQDN) of at least one host system.
```

Comment:
How about we add some very small heading called "Prerequisites" and put the following 3 sentences as bullet points under that Prerequisites heading? It would make it easier to read.

Reply:
That's not a convention that is used in the rest of the doc set. It would add noise imho. Plus I wouldn't advocate for adopting anything from Red Hat's "modular docs".

Reply:
I could go either way on this one. I do see a benefit to a prerequisites' section as it does draw the eye so to speak. But this getting started is written more informally, so it would sort of stick out here.

Reply:
Ok, please **dont implement this suggestion** if it should be inconsistent with the rest of the docs.

In my opinion, the prerequisite section promotes conciseness, ease of reading and as @samccann said it draws the eye a bit more. That's why i proposed this.